docs/console/company-configuration/events.mdx

```diff
@@ -542,235 +1025,4 @@ This event does not guarantee that the user in question already had access to th
 ```
 
 </TabItem>
-</Tabs>
-
-
-## Entities
-
-### Project
```

Comment:
Dov'è stata messa lei?

Reply:
C'è, i due eventi tornano entità diverse di project e quindi ne ho messe due separate in ciascun evento